src/components/calendar.rs

```diff
@@ -0,0 +1,26 @@
+use maud::{html, Markup};
+
+pub fn c() -> Markup {
+    let timeslots: Vec<f64> = Vec::from_iter((17..46).map(|n| n as f64 * 0.5));
```

Comment:
I'm not really sure what that range is, could you elaborate or at least put it in a named variable?

Reply:
Yes, it's supposed to build a vec from 8.5 -> 23.0 at increments of 0.5 to represent the grid on the calendar. I will add a comment to make this clearer (though I'm not sure I'm going to implement the calendar this way; I'm taking a design-as-I-code approach).